core/src/main/java/com/gentics/mesh/core/actions/impl/BranchDAOActionsImpl.java

```diff
@@ -80,7 +80,7 @@ public boolean update(Tx tx, HibBranch branch, InternalActionContext ac, EventQu
 	}
 
 	@Override
-	public void delete(Tx tx, HibBranch element, BulkActionContext bac) {
+	public void delete(Tx tx, HibBranch element) {
```

Comment:
Here and below -- since we consider using BAC from the Tx, we need the explicit BAC arguments no more.